pom.xml

```diff
@@ -64,6 +64,7 @@
         <maven-enforcer-plugin.version>3.0.0-M1</maven-enforcer-plugin.version>
         <maven-jar-plugin.version>3.0.2</maven-jar-plugin.version>
         <maven-surefire-plugin.version>2.21.0</maven-surefire-plugin.version>
+        <maven-surefire-plugin.vmopts>-XX:MaxPermSize=128m</maven-surefire-plugin.vmopts>
```

Comment:
I would also remove maven-surefire-plugin.vmopts and just hardcode to start. We can always add it later if needed.